### PR TITLE
Add egg_locks step to makefile

### PR DESCRIPTION
**Pull request by tconway**

### CODE REVIEW

**Comment by louisnayegon** [comment]
Bamboo job?

**Comment by tconway** [comment]
http://bamboo.digicat.io/browse/CHCL-MPT201-1

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,42 @@
+################################################
+## DO NOT EDIT - AUTOMATICALLY GENERATED FILE ##
+################################################
+astroid==1.4.7
+backports.ssl-match-hostname==3.5.0.1
+git+https://github.com/CDECatapult/behave.git@1.3.1
```
Comment (louisnayegon) [inline requirements.txt]: Problem with the script. This should be
+git+https://github.com/CDECatapult/behave.git@v1.3.1

File: Makefile [hunk 2]
```diff
@@ -59,3 +59,22 @@ behave_system:
...
+		virtualenv $$venv_path ; \
+		source $$venv_path"/bin/activate" ; \
+		pip install -r $$top_level ; \
+		echo "################################################" > $$req_file ; \
+		echo "## DO NOT EDIT - AUTOMATICALLY GENERATED FILE ##" >> $$req_file ; \
+		echo "################################################" >> $$req_file ; \
+		pip freeze >> $$req_file ; \
+		deactivate ; \
+		rm -rf $$venv_path ; \
+		sed -i '' -e 's|behave==\(.*\)|git+https://github.com/CDECatapult/behave.git@\1|g' $$req_file ; \
```
Comment (louisnayegon) [inline Makefile]: Fix
github.com/CDECatapult/behave.git@\1
should be
github.com/CDECatapult/behave.git@v\1

**Pull request merged**